### PR TITLE
Minor changes in C++

### PR DESCRIPTION
**Pull request by RyDroid**

### CODE REVIEW

**Comment by Bertram25** [comment]
Please rebase first. Some of the code has changed.

**Comment by RyDroid** [comment]
I rebased.

File: src/main.cpp [hunk 1]
```diff
@@ -539,14 +539,15 @@ int main(int argc, char* argv[])
     VideoManager->ApplySettings();
 
     // Now the settings are loaded, let's set the windows translated title.
-    /// tr: The window title only supports UTF-8 characters in SDL2.
+    // Translators: The window title only supports UTF-8 characters in SDL2.
```
Comment (Bertram25) [inline src/main.cpp]: Sorry, this is wrong, as @gunchleoc changed the translator comment system in https://github.com/RyDroid/ValyriaTear/commit/26bbbe145825769a7abf52ab5f1a0032c23d8cff
Could you change it back to tr: ?
Thanks

File: src/modes/boot/boot.h [hunk 2]
```diff
@@ -146,7 +146,8 @@ class BootMode : public vt_mode_manager::GameMode
     // ---------- Setup and refresh methods for boot menus
 
     //! \brief Returns whether at least one save is available to be loaded.
-    bool _SavesAvailable();
+    int32_t _NbSavesAvailable();
+    bool    _SavesAvailable();
```
Comment (Bertram25) [inline src/modes/boot/boot.h]: Note for self: I think we can get rid of _SaveAvailable() following your idea which is good, adapt the comment above this function to say it returns the number of available saves, and adapt the client logic.

**Comment by RyDroid** [comment]
I fixed the comment for translation, and write the right documentation for `_NbSavesAvailable` and `_SavesAvailable`, and finally rebased.

**Comment by Bertram25** [comment]
Merged as 96c1eaead258816bd631c01aa833e002f89740aa

**Pull request closed without merging**